llvm/test/Transforms/LoopVectorize/RISCV/remark-reductions.ll

```diff
@@ -0,0 +1,65 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt < %s -mtriple=riscv64 -mattr=+v -p loop-vectorize -pass-remarks-analysis=loop-vectorize -S | FileCheck %s
```

Comment:
This doesn't feel quite right, since you're passing `-pass-remarks-analysis=loop-vectorize` but not testing the output. Given this test is about remarks, I think we should check the output and also ensure we don't see the VF=1 case.

Reply:
Passing `-pass-remarks-analysis=loop-vectorize` just for  triggering  `emitInvalidCostRemarks()` and  make sure the compiler don't crash. I don't have any ideas about testing the remark output.

Something I can imagine is that using  `--debug-only=loop-vecotorize` to print the cost. But it always using the legacy cost model to calculate/print the scalar cost.

Do you have better suggestion to test this case?